### ISSUE TITLE
Currently trying to make this engine work in Unity 2019

### ISSUE THREAD

**Issue by Raychop** [Lallassu/voxelengine_unity#2]
Hello Lallassu,

first of all I want to say that your work is really impressive, and in general your projects are really inspiring me.

For a little presentation, I'm a French Junior Unity Programmer and Game Designer.
Sorry if my english is not that good by the way.

I'm currently trying to make this engine work in Unity 2019, or at least the parts I'm interested in (in the case I can't get everything to work). It's already been a few days I'm working on it.

About the global engine scripts :

• Actually, every shader you made were broken due to the transition from U2017 to U2019, so I swapped them to the very standard unity Shader (making everything white, but at least we can see them. It was broken pink textures before).
It's the main problem of this version actually, as well as lava and pretty much every texture not working correctly.
But, aside this, I fixed some basic compilation problems and all the physics interactions (explosion, etc...) seem to be working perfectly.

About my specific problem :

• I'm making a game where I need to have 3D Voxel environnement + vehicles.
Putting aside the fact I'll need to fix the texture loading problem, I would be incredibly great if you can give me some advice =).
     - 3D Environment : your engine is usable for my project, except for one thing : when loading a voxel model with a "map" type, it is cut into chunks, and all voxel face "looking bellow" (if you see what I mean) are invisible. It will be, of course, quite bad for me considering we can see all the model from all possible angles in my game. I guess you made it on purpose because you are not supposed to see from bellow the map in your engine, but if you know a way to correct this, or where should I look, it would be very appreciated.
     - Vehicles : considering the first point, I wanted my other models, like vehicles, to be instantiated as object alone, because actually it fix this first problem of "missing faces", and it is not sliced into chunk. BUT, here is the problem : .vox files instantiated with the "object" type are not in the world pool, and so can't be destroyed. And I need every vehicules to be destroyable.

** To summarize :

How do you think I/We can make .vox files instantiated with the "object" type physically destroyable like the map ?
OR
How do you think I/We can make .vox files instantiated as "map" to NOT destroy the "bellow faces" ?**

If I can fix one of this 2 issues first, it will be a really great improvement.

To finish, I just want to say that your engine is pretty good, and as a real voxel fan, I think it would be incredibly cool to make it work in Unity 2019 !

In advance, thank's a lot for your answer.
If you want some additionnal info or whatever, feel free to ask me !
I'll also let you know if I get it to work or achieved to adapt it =) .
